addon/components/base/bs-navbar.js

```diff
@@ -53,24 +53,24 @@ import listenTo from 'ember-bootstrap/utils/listen-to-cp';
  prevent that, you can set its `collapseNavbar` property to false:
 
  ```hbs
- {{#nav.item}}
-   {{#nav.link-to "index" collapseNavbar=false}}Don't collapse{{/nav.link-to}}
- {{/nav.item}}
+ <nav.item>
+  <nav.link-to "index" @collapseNavBar={{false}}>Don't collapse</nav.link-to>
+ </nav.item>
  ```
 
  To collapse the navbar when clicking on some nav items that are not internal links, you can use the yielded `collapse`
  action:
 
  ```hbs
- {#bs-navbar as |navbar|}}
-   {{#navbar.content}}
-     {{#navbar.nav as |nav|}}
-       {{#nav.item}}
-         <a {{action navbar.collapse}}>Collapse</a>
-       {{/nav.item}}
-     {{/navbar.nav}}
-   {{/navbar.content}}
- {{/bs-navbar}}
+ <BsNavbar as |Navbar|>
+  <Navbar.content>
+    <Navbar.nav as |Nav|>
+      <Nav.item>
+        <a {{action Navbar.collapse}}>Collapse</a>
```

Comment:
I would prefer closure actions over element modifiers. But I'm not sure what should be considered current best practice.

Reply:
Agreed.

https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/rule/no-action-modifiers.md

Reply:
Though this rule is not with the `recommended` set! But I see you already updated that, so that's fine!